src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -24,13 +24,15 @@ ci_lazy_methodst::ci_lazy_methodst(
   const std::vector<irep_idt> &main_jar_classes,
   const std::vector<irep_idt> &lazy_methods_extra_entry_points,
   java_class_loadert &java_class_loader,
+  const std::vector<irep_idt> &java_load_classes,
```

Comment:
I realise no docs above, but could you document this parameter (leaving the rest blank is fine)

Reply:
@thk123 done

Reply:
Suggested name: `extra_needed_classes`, since they will be considered "needed" (available, instantiated) classes in the same way as `needed_classes` below.